src/track/bpm.cpp

```diff
@@ -38,6 +38,7 @@ double Bpm::valueFromString(const QString& str, bool* pValid) {
 
 QString Bpm::valueToString(double value) {
     if (isValidValue(value)) {
+        //TODO: Shouldn't this be formatted in some way, instead of letting it output in scientific notation?
```

Comment:
You might add a comment that this non-localized(!) format is the default for most tag types. Special cases like rounding to integer for ID3v2 are handled separately.

Formatting for display purposes is handled by Track::getBpmText(). Currently this doesn't honour the user's locale settings, see also QLocale::toString(). But before modifying the current behaviour we need to check how manual inputs are handled.